### ISSUE TITLE
Why use minimal action set and not legal action set like in Revisiting ALE paper?

### ISSUE THREAD

**Issue by marintoro** [google/dopamine#101]
I think this is an issue for reproducibility and comparaison.
The paper "Revisiting ALE [...]" by Machado et al. (2018) use the full action set (18 actions) on all games and it's hard to know what is exactly the impact of giving MinimalActionSet compared to FullActionSet.

I know that this was an issue to reproduce Breakout result:
"DeepMind used a minimal set of four actions in Breakout (xitari), several versions of OpenAi gym’s Breakout have six actions. Additional actions can alter the difficulty of the task the agent is trying to learn drastically! The Breakout-v4 and BreakoutDeterministic-v4 environments have four actions (check with env.unwrapped.get_action_meanings())." (from [this blog](https://towardsdatascience.com/tutorial-double-deep-q-learning-with-dueling-network-architectures-4c1b3fb7f756))

**Comment by mgbellemare** [comment]
Fair point, in hindsight minimal actions were a mistake from day one. IIRC it was more difficult to use the full action set when operating via Gym. It would also make the results less comparable to what's already published (but we've pushed this with enabling sticky actions anyway).
